src/consensus/tx_verify.cpp

```diff
@@ -27,6 +29,43 @@ bool IsFinalTx(const CBaseTransaction &tx, int nBlockHeight, int64_t nBlockTime)
     return true;
 }
 
+
+bool IsLicenseTx(const CBaseTransaction& tx) {
+	if (tx.IsNull() || tx.IsCoinBase())
+		return false;
+
+	if (tx.vin.size() != 1)
+		return false;
```

Comment:
Sometimes it might be necessary to spend more than one UTXO. We should not impose this condition, but rather check every input.

Reply:
Done